frontend/components/App/App.tsx

```diff
@@ -49,6 +52,10 @@ const App = ({ children, router }: IAppProps): JSX.Element => {
         setCurrentUser(user);
         setAvailableTeams(available_teams);
       } catch (error) {
+        console.error(error);
+        if (!location || location?.pathname === "/setup") {
+          return;
```

Comment:
```suggestion
localStorage.removeItem("auth_token");
return;
```

Reply:
The only change I'd suggest is going ahead and removing the localStorage token since we know it's invalid.